net/third_party/quiche/src/quiche/quic/test_tools/send_algorithm_test_utils.cc

```diff
@@ -35,8 +35,11 @@ void RecordSendAlgorithmTestResult(uint64_t random_seed,
 void CompareSendAlgorithmTestResult(int64_t actual_simulated_duration_micros) {
   SendAlgorithmTestResult expected;
   ASSERT_TRUE(LoadSendAlgorithmTestResult(&expected));
+#ifndef USE_COBALT_CUSTOMIZATIONS
+// Cobalt's protobuf is too old to support ShortDebugString().
```

Comment:
Do we need to update it ?

Reply:
It would be good to do for better compatibility i.e. here but I don't believe it blocks the base/net update